bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -480,7 +486,7 @@ func Run(opt *options.ServerOption) error {
 			log.Infof("Components applied: " + out.String())
 		}
 	}
-	if opt.InCluster {
+	if opt.InCluster && opt.KeepAlive {
```

Comment:
Shouldn't we keepAlive when `opt.KeepAlive` is True regardless of what `opt.InCluster` is set to

Reply:
When ```opt.InCluster=false```, bootstrapper is not executed inside a pod, then we don't need to prevent pod from exit. Likely that user is using bootstrapper in interactive mode. (Local docker)